lib/go-tc/servers.go

```diff
@@ -574,11 +577,96 @@ func (s Server) ToNullable() ServerNullableV2 {
 	}
 }
 
+// ToNonNullable converts the ServerNullableV2 safely to a Server structure.
+func (s ServerNullableV2) ToNonNullable() Server {
+	coerceBool := func(b *bool) bool {
+		if b == nil {
+			return false
+		}
+		return *b
+	}
+	coerceInt := func(i *int) int {
+		if i == nil {
+			return 0
+		}
+		return *i
+	}
+	coerceString := func(s *string) string {
+		if s == nil {
+			return ""
+		}
+		return *s
+	}
```

Comment:
These functions should be declared at the package level for reuse. Go intentionally disallows declaring functions within other functions.